src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from 'classnames';
+import { useAccount } from 'wagmi';
+import {
+    DataList,
+    NumberFormat,
+    Tag,
+    formatterUtils,
+    type IDataListItemProps,
+    type TagVariant,
+} from '../../../../../core';
+import { type ICompositeAddress } from '../../../../types';
+import { addressUtils } from '../../../../utils';
+import { MemberAvatar } from '../../../member';
+
+export interface IVoteDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The account details of the voter.
+     */
+    voter: ICompositeAddress;
+    /**
+     * Whether the voter is a delegate of the current user or not.
+     */
+    isDelegate?: boolean;
+    /**
+     * The vote of the user.
+     */
+    voteIndicator: 'yes' | 'no' | 'abstain' | 'approved';
+    /**
+     * If token-based voting, the amount of token voting power used.
+     */
+    votingPower?: number | string;
+    /**
+     * If token-based voting, the symbol of the voting power used.
+     */
+    tokenSymbol?: string;
+}
+
+const voteIndicatorToTagVariant: Record<IVoteDataListItemStructureProps['voteIndicator'], TagVariant> = {
+    yes: 'success',
+    no: 'critical',
+    abstain: 'neutral',
+    approved: 'primary',
+};
+
+export const VoteDataListItemStructure: React.FC<IVoteDataListItemStructureProps> = (props) => {
+    const { voter, isDelegate, votingPower, tokenSymbol, voteIndicator, ...otherProps } = props;
+    const { address: currentUserAddress, isConnected } = useAccount();
+
+    const isCurrentUser = isConnected && currentUserAddress === addressUtils.getChecksum(voter.address);
```

Comment:
Could you use the `isAddressEqual` utility to compare addresses here and update the memberDataListItem component as well? 🙏🏻